config/tailwind/v3.x.x/tailwind.config.js

```diff
@@ -0,0 +1,270 @@
+// Import GC Design System tokens
+import tokens from "./node_modules/@cdssnc/gcds-tokens/build/tailwind/tailwind.tokens.json";
```

Comment:
From my testing it looks like we could remove the `./node_modules/` in the imports. Cleans it up a bit.

```suggestion
import tokens from "@cdssnc/gcds-tokens/build/tailwind/tailwind.tokens.json";
```